src/legacy/core_plugins/kibana/public/discover/angular/discover.js

```diff
@@ -151,7 +146,7 @@ uiRoutes
         return savedSearches.get(savedSearchId)
           .then((savedSearch) => {
             if (savedSearchId) {
-              npStart.core.chrome.recentlyAccessed.add(
+              chrome.recentlyAccessed.add(
```

Comment:
dear @flash1293, `savedSearches` is the reason why this currently doesn't work, can't see the wood for the trees here, so I'd appreciate you input.

Reply:
You should make `savedSearches` part of kibana_services and pass it in from the outer angular - check out the euqivalent in Graph: https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/graph/public/index.ts#L42

Migrating the `SavedObject` hierarchy is on our roadmap anyway, so we can treat this as an external service for now and pull it in in a second step.